src/nile/core/account.py

```diff
@@ -26,10 +26,24 @@
 load_dotenv()
 
 
-class Account:
+class AsyncObject(object):
+    """Base class for Account to allow async initialization."""
+
+    async def __new__(cls, *a, **kw):
+        """Return coroutine (not class so sync __init__ is not invoked)."""
```

Comment:
this might not be necessary if we advance with https://github.com/OpenZeppelin/nile/issues/278. maybe worth adding a comment about it